components/combobox/css/index.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021-2022 Gemeente Utrecht
+ * Copyright (c) 2021-2022 Frameless B.V.
+ */
+
+@import "./mixin";
+
+.utrecht-combobox {
+  @include utrecht-combobox;
+}
+
+.utrecht-combobox_list {
+  @include utrecht-combobox_list;
+}
+
+.utrecht-combobox_list-item {
+  @include utrecht-combobox_list-item;
+}
+
+.utrecht-combobox_list-item:hover {
+  @include utrecht-combobox_list-item--hover;
+}
+
+.utrecht-combobox_list-item[aria-selected="true"] {
+  @include utrecht-combobox_list-item--highlighted;
+}
+
```

Comment:
Wat is het verschil tussen `selected` en `highlighted`?

Reply:
highlighted is when you walk through the list by using the keyDown or KeyUp
Selected when you select an element, it adds  style to the selected element when you reopen the list again